com.trollworks.gcs/src/com/trollworks/gcs/settings/DamageProgression.java

```diff
@@ -131,6 +131,29 @@ public Dice calculateThrust(int strength) {
         public Dice calculateSwing(int strength) {
             return BASIC_SET.calculateSwing(strength);
         }
+    },
+    SWING_EQUALS_THRUST_PLUS_2 {
+        @Override
+        public String toString() {
+            return I18n.text("Swing = Thrust+2");
+        }
+
+        @Override
+        public String getFootnote() {
+            return I18n.text("Houserule originally originating with Kevyn Smith");
```

Comment:
Please change "originally originating" to just "originating"

Reply:
Done; I've also added a link to the session recap and fixed Kevin's name.